res/controllers/midi-components-0.1.js

```diff
@@ -0,0 +1,1135 @@
+"use strict";
+/**
+ * Components JS library for Mixxx
+ * Documentation is on the Mixxx wiki at
+ * http://mixxx.org/wiki/doku.php/components_js
```

Comment:
this link is broken, should be https://github.com/mixxxdj/mixxx/wiki/Components-JS